docs/dev-proc/standard-release-process.md

```diff
@@ -0,0 +1,53 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged to
+   into main.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Create and push tag that matches the newly bumped version.
+4. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
+      recipe versions. The script uses build manifests - lists of recipes
+      contained in the image along with their versions. You must provide your
+      manifest and the current release's manifest as arguments to the script.
+      The latter can be found in the latest release's section in the GitHub
+      release pages.
+    - Paste the script output into your release's section in `CHANGELOG.md`.
+      Additionally, you must explain why those changes were made and what they
+      introduce (unless the commit messages listed by `generate-changelog.sh`
+      already explain that).
+    - Describe any additional changes.
+5. Commit and push the changelog.
+6. Publish the release on GitHub. Make sure to upload the manifest.
+
+## Versioning scheme
+
+`Zarhus`-related layers are versioned using the
+[semantic versioning](https://semver.org) scheme with the following
+clarifications and additions:
+
+- `PATCH` version is incremented with each released `HOTFIX`
+- `PATCH` version is zeroed with the increment of `MINOR` or `MAJOR` versions
+- `MINOR` version is incremented with every `FEATURE` release (may include more
+  than one feature) unless `MAJOR` version is incremented
+- `MINOR` version is zeroed with the increment of `MAJOR` version
+- `MAJOR` version is incremented according to the project road map. Project road
+  map should define at which point in time or with which feature set next
+  `MAJOR` release is ready.
+
+Depending on the need of a particular software project, a `pre-release`
+(`release candidate`) version may be released. The version format must follow
+the [pre-release semver versioning scheme](https://semver.org/#spec-item-9).
+Generally, we shall use the following scheme for `release candidates`:
+
+```
```

Comment:
Specify language. You can add `text` or anything.
If we add pre-commit in the future with `markdownlint` then this line would error:

```text
dev-proc/standard-release-process.md:49 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
```

Reply:
I always disliked this rule. What language should I specify here? I can write something like "text", but maybe we should discuss if we should disable the rule in our pre-commits. It's really annoying in cases like this one, where it's not actually a block of code.

Reply:
https://github.com/jincheng9/markdown_supported_languages

> * text ('*.txt')

Reply:
Another example:

https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md

> plaintext, txt, text

That's why I think specifying `text` is ok for text blocks

Reply:
https://github.com/zarhus/docs/compare/7e49970ef26a4ad75c1583d743e66fbb7f7de3b2..3340801d827e38a8c37ea7af5777b95a9b437c1a#diff-601babdf30d07c9e8dc632e099bf77c53afe5e0ec2dc70a2d37ea0cbfdbe3c5cR49